LuisPalominoTrevilla/Guessit-back
# Feat/GI-19/UpdateProfile

## Changes:
- Added UpdateProfile endpoint to allow user to edit their data.

## Notes:
- It's still missing documentation, will fix that later.
- I wasn't sure which values you could edit, so I started with the basic ones. Let me know if we should modify something else.

## How to test:
- Using postman (or a similar program used to make requests), send a PATCH request to Localhost:5000/api/User/Update containing a JSON with the email of the account that wants to update data and the new image path, username, and/or password. See image below for reference.
<img width="857" alt="Screen Shot 2019-05-05 at 10 28 22 PM" src="https://user-images.githubusercontent.com/31036932/57205571-20045880-6f85-11e9-8f18-0c87b7bf1120.png">
